pkg/controller/config/config_controller.go

```diff
@@ -152,34 +165,46 @@ func (r *ReconcileConfig) Reconcile(req reconcile.Request) (reconcile.Result, er
 		return reconcile.Result{}, err
 	}
 
-	if isUpToDate(cfg) {
+	if stateCheck(cfg, op.InstalledStatus) {
```

Comment:
How about calling it `isStatus(cfg. op.InstalledStatus)` also how about `cfg.IsStatus(op.InstalledStatus)` ?